gatsby_fp/package.json

```diff
@@ -31,7 +31,7 @@
     "gatsby-plugin-react-helmet": "^3.3.3",
     "gatsby-plugin-sharp": "^2.6.10",
     "gatsby-source-filesystem": "^2.3.10",
-    "gatsby-source-multi-api-FORKED": "git+https://github.com/Stevens-EWS/gatsby-source-multi-api.git#8166ab2632dfb16bcfef95d9de244866141de6b2",
```

Comment:
This has nothing to do with making the logo a link. Something got messed up with this PR